main/inc/introductionSection.inc.php

```diff
@@ -290,6 +290,13 @@
         api_get_path(WEB_CODE_PATH).'course_info/tools.php?'.api_get_cidreq(),
         ['class' => 'btn btn-default', 'title' => get_lang('CustomizeIcons')]
     );
+
+}
+/* Tool to show /hide all tools on course */
+$toolAllShowHide = '';
+if (api_is_allowed_to_edit() && empty($session_id)) {
+    $toolAllShowHide =  '<button class="btn btn-default hidden visible-all  show-hide-all-tools" title="'.get_lang('Activate', '').'"><em class="fa fa-eye "></em></button>';
+    $toolAllShowHide .=  '<button class="btn btn-default hidden invisible-all show-hide-all-tools" title="'.get_lang('Deactivate', '').'"><em class="fa fa-eye-slash"></em></button>';
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 182 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5714830)